assets/overlays/openstack-manila/patches/node_add_driver.yaml

```diff
@@ -86,6 +92,18 @@ spec:
               memory: 50Mi
           terminationMessagePolicy: FallbackToLogsOnError
       volumes:
+        - name: config-manilaplugin
+          configMap:
+            name: cloud-conf
+            items:
+              - key: cloud.conf
+                path: cloud.conf
+        - name: secret-manilaplugin
```

Comment:
I know you're aligning with Cinder again here, but would a more descriptive name be helpful? Say, `cloud-credentials`? Again, I'm planning on doing the same thing for Cinder.